services/src/api/handlers/wcs.rs

```diff
@@ -430,6 +430,7 @@ async fn wcs_get_coverage_handler<C: ApplicationContext>(
         spatial_bounds: request_partition,
         time_interval: request.time.unwrap_or_else(default_time_from_config).into(),
         spatial_resolution,
+        selection: Default::default(), // TODO
```

Comment:
do what?

Reply:
I made it more conrete e6b9479658da849934df78cc2b2b328aad6511b9